pytorch_toolkit/image_classification/model_templates/custom-classification/README.md

```diff
@@ -1,15 +1,78 @@
 # Image classification
 
-Models that are able to classify images.
-
-| Model Name | Complexity (GFLOPs) | Size (Mp) | Top-1 on ImageNet (%) | Links |
-| ---        | ---                 | ---       | ---                   | ---                   |
-| MobilenetV2 1.0 | 0.66           | 3.50      | 73.0                  | [model template](./mobilenet_v2_w1/template.yaml)
-
-## Datasets
-
-The model was initially trained on [ImageNet](http://image-net.org/challenges/LSVRC/2012/), but can be trained from scratch or fine tuned to classify arbitrary images.
-
+Performance results pivot table:
+
+| Model Name | Complexity (GFLOPs) | Size (Mp) | AVG mAP (%) | AVG Top-1 (%) | AVG Top-5 (%) | Links |
+| ---        | ---        | ---        | ---        | ---        | ---        | ---        |
+| EfficientNet b0 | 0.76 | 4.14 | 92.267 | 90.156 | 98.283 | [model template](./efficientnet_b0/template.yaml) |
+| MobilenetV3 large x1.0 | 0.44 | 4.33 | 92.04	| 89.019 | 97.807 | [model template](./mobilenet_v3_large_1/template.yaml) |
+| MobilenetV3 large x0.75 | 0.308 | 2.84 | 87.59 |	87.59	| 97.17 | [model template](./mobilenet_v3_large_075/template.yaml) |
+| MobilenetV3 small x1.0 | 0.112 | 1.56 | 82.84 | 84.47 | 95.88 | [model template](./mobilenet_v3_small/template.yaml) |
+
+All of the above metrics were obtained on eleven different datasets, on which an extensive number of researches has been made.
+
+The following datasets were used in experiments:
+* [Describable Textures (DTD)](https://www.robots.ox.ac.uk/~vgg/data/dtd/)<sup>1</sup>
+* [Caltech 101](http://www.vision.caltech.edu/Image_Datasets/Caltech101/)<sup>1</sup>
+* [Oxford 102 Flowers](http://www.robots.ox.ac.uk/~vgg/data/flowers/102/)
+* [Oxford-IIIT Pets](https://www.robots.ox.ac.uk/~vgg/data/pets/)
+* [CIFAR100](https://www.cs.toronto.edu/~kriz/cifar.html)
+* [SVHN (w/o additional data)](http://ufldl.stanford.edu/housenumbers/)
+* [Fashion MNIST](https://github.com/zalandoresearch/fashion-mnist)
+* [FOOD101](https://www.kaggle.com/dansbecker/food-101)<sup>1</sup>
+* [SUN397](https://vision.princeton.edu/projects/2010/SUN/)<sup>1</sup>
+* [Birdsnap](http://thomasberg.org/)<sup>1</sup>
+* [Cars Dataset](https://ai.stanford.edu/~jkrause/cars/car_dataset.html)
+
+<sup>1</sup> these datasets have custom splits (random stratified split: 80% - train, 20% - val) and cannot be compared straightforward with other research results
```

Comment:
I'm not sure, but it seems that  `cannot be compared straightforward` -> `cannot be compared straightforwardly`